ec/xyzz_t.hpp

```diff
@@ -116,7 +116,7 @@ class xyzz_t {
     };
 
     class affine_inf_t { friend class xyzz_t;
-        field_h X, Y;
+        field_t X, Y;
 #ifdef __CUDACC__
         int inf[sizeof(field_t)%16 ? 2 : 4];
```

Comment:
Hmmm... The reason for why affine_inf_t was available only in "h" flavour, a.k.a. field_t::mem_t is because  this `inf` field is explicitly "mem_t". Of course  the thing here is that **we** ourselves don't use the affine_inf's "t" flavour, so it doesn't cause problem for **us**, but if it will be used for whatever reason in the future, then this will fail. For this reason it's appropriate to resolve it now:-) The right thing to do is to declare `inf` bool unconditionally, and add method to convert affine_int_t::mem_t to affine_inf_t that would load the value from memory.

Reply:
If you want, I can fix it up myself afterwards...

Reply:
> ... declare `inf` bool unconditionally, and ... load the value from memory.

Just in case, using the is_inf() method. As in `inf = p.is_inf()` as opposed to `inf = p.inf[0]`.

Reply:
Will the last commit do?